web/templates/blocks/clusters_table.html.tmpl

```diff
@@ -15,17 +15,19 @@
             </thead>
             <tbody>
             {{- range . }}
-                <tr>
+                <tr id="cluster-{{ .ID }}" class="cluster-{{ .Name }}">
                     <td class="row-status">{{ template "health_icon" .Health }}</td>
                     <td>
                         {{- if .HasDuplicatedName }}
                             <i class="eos-icons eos-18 text-info" data-toggle="tooltip" data-original-title="This cluster has a duplicated name">info</i>
                         {{- end }}
+                        <span class="tn-clustername">
```

Comment:
This `span` looks a bit suspicious. We have not used in any of the other links in tables.
If you need the class, why don't simply add it to the `a` element.

Reply:
Yep, it's ugly, it's there because of the `{{- if ne .ClusterType "Unknown" }}`
that in case of an unknown cluster type, it wouldn't add the `<a>` element to the DOM. I wanted to keep changes to the templates minimal as it's outside the scope of the PR but we can improve this in the future for sure